app/src/main/res/layout/activity_health_macro_tracker.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/colorHealthHome"
+    android:gravity="center_horizontal"
+    android:orientation="vertical"
+    android:padding="16dp"
+
+    tools:context=".MainActivity">
+
+    <TextView
```

Comment:
We want to make sure we include the `hint` attribute for the phone's screen reader. Also want to keep any string that will be displayed to the user in the `values/strings` file so everything can be easily translated if we choose that option down the road.